### DISCUSSION TITLE
Island Upgrade Price Placeholder

### DISCUSSION THREAD

**Discussion by Silarious** [BG-Software-LLC/SuperiorSkyblock2#835] [Pending Feature Requests] [answered]
### Summary
`%superior_island_upgrade_price_<upgrade>-<level>%`
Reports price of the specified upgrade level.

## Reason
Removes the need to manually update prices in upgrade menu, especially important cause of lore being in both no-next-level and has-next-level.
This would also open up displaying island upgrades and their prices in other menus, command responses, actionbar, scoreboards, etc.

**Comment by OmerBenGera** [comment]
I dont think its necessary.
Giving you the ability to format the price however you want will not be possible using a placeholder, for example.
Also, I dont see any advantages of using a placeholder over a regular text.

**Reply by Silarious** [reply]
I dont quite understand what you mean by "format the price however you want" all the placeholder would do is output the price of the specified upgrade level as float or int just like how its input.

```
generator-rates:
  '1':
    price: 2000
```
%superior_island_upgrade_price_generator-rates-1% -> 2000

Totally understand if this is not possible.

**Reply by OmerBenGera** [reply]
> I dont quite understand what you mean by "format the price however you want" all the placeholder would do is output the price of the specified upgrade level as float or int just like how its input.
> 
> ```
> generator-rates:
>   '1':
>     price: 2000
> ```
> 
> %superior_island_upgrade_price_generator-rates-1% -> 2000
> 
> Totally understand if this is not possible.

Many prefer having the prices formatted, similar to how they are formatted in the default files - commas, etc.

**Comment by Lemvs** [comment, marked as answer]
https://github.com/BG-Software-LLC/SuperiorSkyblock2/discussions/135#discussioncomment-17672431

@OmerBenGera you can mark this answered and close it.
